database/factories/UserFactory.php

```diff
@@ -58,6 +58,13 @@ public function definition()
             'accepted_cookie_policy_at' => now(),
             'preferred_foot' => $this->faker->randomElement(['left', 'right']),
             'rating' => $this->faker->randomFloat(2, 0, 5),
+            'last_known_ip' => $this->faker->ipv4(),
+            'current_country' => $this->faker->country(),
+            'current_region' => $this->faker->region,
```

Comment:
Make this a method not an attribute faker attributes will be deprecated in future versions.